api/provider/aws/environment_create.go

```diff
@@ -81,39 +49,69 @@ func (e *EnvironmentProvider) Create(req models.CreateEnvironmentRequest) (strin
 		return "", err
 	}
 
-	launchConfigName := fqEnvironmentID
-	if err := e.createLC(
-		launchConfigName,
-		aws.StringValue(securityGroup.GroupId),
-		instanceType,
-		e.Config.InstanceProfile(),
-		e.Config.SSHKeyPair(),
-		amiID,
-		userData); err != nil {
-		return "", err
-	}
+	// creating asg, lc isn't required for dynamic environments
```

Comment:
[nit] for this comment, took a minute for me to realize lc == launch config 🤦‍♂️